audmetric/core/api.py

```diff
@@ -114,20 +120,37 @@ def concordance_cc(
     if len(prediction) < 2:
         return np.NaN
 
-    r = pearson_cc(prediction, truth)
-    x_mean = prediction.mean()
-    y_mean = truth.mean()
-    x_std = prediction.std()
-    y_std = truth.std()
-    denominator = (
-        x_std * x_std
-        + y_std * y_std
-        + (x_mean - y_mean) * (x_mean - y_mean)
-    )
+    # Handle mask NaN cases separetly
+    # to be as fast as possible
+    consider_nan = False
+    if ignore_nan:
+        mask = ~(np.isnan(truth) + np.isnan(prediction))
```

Comment:
```suggestion
        mask = ~(np.isnan(truth) | np.isnan(prediction))
```

Seems more natural to me.